howto/manage-groups.md

```diff
@@ -4,29 +4,34 @@
 >
 > See also: {ref}`group`
 
+```{note}
+This guide assumes you have the `jaas` plugin installed.
```

Comment:
I would skip this note. Otherwise we'd have to put it everywhere, which feels wrong.